internal/util/clock/ticker.go

```diff
@@ -0,0 +1,25 @@
+package clock
+
+import "time"
+
+func NewTicker() *TimeTicker {
+	return &TimeTicker{
+		ticker: time.NewTicker(1000 * time.Hour),
```

Comment:
Would you mind adding a small comment or moving to a constant with a meaningful name the `1000 * time.Hour` expression? It's not obvious for reader why we set this like that